jquery.pjax.js

```diff
@@ -310,9 +316,11 @@ function pjax(options) {
     // the last field.
     //
     // http://www.w3.org/html/wg/drafts/html/master/forms.html
-    var autofocusEl = context.find('input[autofocus], textarea[autofocus]').last()[0]
-    if (autofocusEl && document.activeElement !== autofocusEl) {
-      autofocusEl.focus();
+    if(blurFocus) {
```

Comment:
If an autofocus element exists within the new pjax content, it should be autofocused unconditionally. This is the old behavior, and I don't think it should be changed. I wouldn't add this conditional here.